queries/elixir/aerial.scm

```diff
@@ -25,6 +25,17 @@
   (#set! "kind" "Function")
   ) @type
 
+(call
+  target: (identifier) @identifier (#any-of? @identifier "defp" "defmacrop" "defguardp")
+  (arguments [
+              (call target: (identifier) @name)
+              (binary_operator left: (call target: (identifier) @name))
+              ((identifier) @name)
+   ])
+  (#set! "kind" "Function")
+  (#set! "scope" "private")
+  ) @type
+
```

Comment:
I think adding the scope with `#set!` is a good additional capability, but for the elixir case I feel like we should use the previous extension-based logic. It's so much cleaner to have a single if-statement to set the scope than to duplicate this entire query